pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -200,13 +247,28 @@ func (r *ReconcileNodeMaintenance) Reconcile(request reconcile.Request) (reconci
 		return r.reconcileAndError(instance, err)
 	}
 
+	if r.isLeaseSupported {
+		reqLogger.Info("Lease object supported, obtaining lease")
+
+		proceed, reconcileRes := r.obtainLease(node, LeaseDurationInSeconds, reqLogger)
```

Comment:
change reconcileRes to either result or reconcileResult  for a meaningful name with no abbreviations .